index/traces-relational-watts.txt

```diff
@@ -6,8 +6,8 @@ base-mm: --set ana.base.privatization mutex-meet
 base-mm-interval: --set ana.base.privatization mutex-meet --enable ana.int.interval
 box: --set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.apron.privatization mutex-meet --sets ana.apron.domain interval
 oct: --set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.apron.privatization mutex-meet
-tid: --set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.apron.privatization mutex-meet-tid
-cluster12: --set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.apron.privatization mutex-meet-tid-cluster12
+tid: --set ana.activated[+] apron --set ana.activated[+] threadJoins --set ana.path_sens[+] threadflag --set ana.apron.privatization mutex-meet-tid
+cluster12: --set ana.activated[+] apron --set ana.activated[+] threadJoins --set ana.path_sens[+] threadflag --set ana.apron.privatization mutex-meet-tid-cluster12
```

Comment:
This is in the watts index, not ratcop, but we should probably add threadJoins to all of them to rerun the benchmarks and see if it makes any difference whatsoever.

Reply:
I now also changed the ratcop index. I decided to not revert the commit for Watts as it should be changed, anyways.